components/screens/AddWallet.tsx

```diff
@@ -60,17 +61,17 @@ export default function AddWallet() {
         </button>
 
         {/* Ledger */}
-        {/* TODO: Enable it after the UI is ready
-        <button
-          className="flex items-center gap-2 rounded-full bg-icy-blue-400 p-5"
-          onClick={() => {
-            browser.tabs.create({ url: "/popup.html#/import-ledger" });
-          }}
-        >
-          <span className="text-base">Import with Ledger</span>
-          <i className="hn hn-arrow-right flex-none text-[14px]"></i>
-        </button>
-        */}
+        {settings?.preview && (
+          <button
+            className="flex items-center gap-2 rounded-full bg-icy-blue-400 p-5"
+            onClick={() => {
+              browser.tabs.create({ url: "/popup.html#/import-ledger" });
+            }}
+          >
+            <span className="text-base">Import with Ledger</span>
+            <i className="hn hn-arrow-right flex-none text-[14px]"></i>
+          </button>
+        )}
       </div>
     </div>
```

Comment:
Can you enable ledger for dev mode in other PR? I realize that we need to check HID devices permissions about app store.

Reply:
No worries

Reply:
1911b1de6a38b9ff721549f2f4d1624b7b292fb9